classes/Console/Command/AdminDemoteCommand.php

```diff
@@ -33,13 +28,6 @@ protected function configure()
 );
     }
 
-    /**
-     * Execute the command.
-     *
-     * @param  \Symfony\Component\Console\Input\InputInterface   $input
-     * @param  \Symfony\Component\Console\Output\OutputInterface $output
-     * @return void
```

Comment:
This is actually outdated, too. 

Running

```
$ composer show | grep symfony/console
```

yields

```
symfony/console                   v2.8.0             Symfony Console Component
```

See https://github.com/symfony/console/blob/v2.8.0/Command/Command.php#L158.